src/odemis/gui/main.xrc

```diff
@@ -5582,7 +5582,7 @@
                                                     <object class="UnitFloatCtrl" name="param_Zmax">
                                                         <size>-1,15</size>
                                                         <value>1</value>
-                                                        <key_step>0.1</key_step>
+                                                        <key_step>0.000001</key_step>
                                                         <min>0</min>
                                                         <max>1000</max>
```

Comment:
The min/max of these controls don't match the VAs. You could either hard-code the same value here (as in the gui.model.__init__), or when connecting them, update the range of the UnitFloatCtrl using `.SetValueRange()`.